### ISSUE TITLE
[enhanced-parts] DDC implementation

### ISSUE THREAD

**Issue by eernstg** [dart-lang/sdk#56197]

**Comment by davidmorgan** [comment]
@bkonyi @nshahan @Markzipan did anyone ever look into DDC work for Parts With Imports (previously known as Enhanced Parts)?

**Comment by nshahan** [comment]
@davidmorgan 
AFAIK we have been waiting for a signal from the Model team that work in the kernel AST is completed and we should begin reviewing test results. Can we consider your ping that signal? Is there a language test suite we should take a look at?

I'd imagine this feature is intended as being handled by the kernel with little to no implementation needed in the backends. That said, there could always be some unexpected edge cases whenever a representation changes so we should look out for some work being needed.

**Comment by davidmorgan** [comment]
Yes, the CFE implementation is feature complete, so it'd be a good time to take a first look.

co19 tests are in, Language tests are pending

https://github.com/dart-lang/sdk/issues/56191
https://dart-review.googlesource.com/c/sdk/+/452101
